src/Radicle/Ipfs.hs

```diff
@@ -214,11 +214,13 @@ subscribe :: (MonadIpfs m) => Text -> (PubsubMessage -> IO ()) -> m ()
 subscribe topic messageHandler = do
     Client{clientHttpManager, clientBaseRequest} <- askClient
     liftIO $ runResourceT $ do
-        let req = clientBaseRequest
-                & HttpClient.setQueryString
-                    [ ("arg", Just $ T.encodeUtf8 topic)
-                    , ("encoding", Just "json")
-                    , ("stream-channels", Just "true")
+        let req = makeIpfsRequest
+                    clientBaseRequest
+                    Http.methodGet
+                    "pubsub/sub"
```

Comment:
Am I reading this right that before we just made a request to the root URL and this never actually worked?

Reply:
Not in the past 20 days...